src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs

```diff
@@ -1018,18 +1018,24 @@ private void SetSplicer()
 
 		private void DoTriggeredAutoRestoreIfNeeded()
 		{
+			// Disable the seek that could have been initiated when painting.
+			// This must done before DoAutoRestore, otherwise it would disable the auto-restore seek.
+			if (_playbackInterrupted)
+			{
+				MainForm.PauseOnFrame = null;

```

Comment:
Desired behavitor (the way taseditor does it):
Disable autorestore, pause, edit, unpause to start seeking: then no matter how much you edit _while it's still seeking_, it will pause on the seek frame.

With this edit, it won't pause there in this situation, but I'm not sure when this started or if it ever worked like in taseditor.